### PR TITLE
fix:changed to grid layout

### PR DESCRIPTION
**Pull request by varun-Tad**
Created images and inputs

### CODE REVIEW

**Comment by gauravsinhaweb** [comment]
https://user-images.githubusercontent.com/75125943/152631913-b7968d1d-fbc2-42d6-aa2a-1e497d5f3426.mp4

the website zoom in, in mobile responsive, to avoid that you can add content width to 100%. ` <meta name="viewport" content="width=100%" />`


nice work done!

File: Components/Alerts/alerts.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .alert{
-    width:300px;
+    width:30rem;
     text-align: center;
```
Comment (Ankur9669) [inline Components/Alerts/alerts.css:3]: You could use max-width here than your alert would become responsive. But that's just a suggestio this also looks fine.

File: Components/Badges/badge.css [hunk 2]
```diff
@@ -5,55 +5,55 @@
...
+    height:2rem;
     border-radius: 50%;
     font-weight:700;
     position:relative;
-    bottom:25px;
-    left:70px;
+    bottom:2.5rem;
+    left:7rem;
     
 }
```
Comment (Ankur9669) [inline Components/Badges/badge.css:54]: image
1. See you are repeating the code border-radius 50% AND font-weight so you can creat a class 
badge{
    border-radius: 50%;
    font-weight:700;
}
And use it everywhere. This also looks good by the way

File: Components/Buttons/buttons.css [hunk 3]
```diff
@@ -177,7 +177,7 @@
     color:var(--solid-btn-text-color);
     color:var(--secondary-color);
     background-color: black;
-    padding:15px;
```
Comment (Ankur9669) [inline Components/Buttons/buttons.css:180]: Try to use css variable here

File: Components/Buttons/buttons.css [hunk 4]
```diff
@@ -177,7 +177,7 @@
...
+    padding:1.5rem;
     cursor: pointer;
     border:none;
 }
```
Comment (Ankur9669) [inline Components/Buttons/buttons.css:183]: image
You have used hover properties but see all the hover are using same opacity:0.7
So you can just do 
.btn:hover{
     opacity:0.7
}
And then you can safely remove other classes

File: Components/Image/image.css [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  }
+
+
+  .round-img{
+    height: 500px;
+    width:500px;
+    border-radius: 50%;
```
Comment (Ankur9669) [inline Components/Image/image.css:22]: You can give rems here if you want

File: Components/Input/input.css [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+    background-color: var(--primary-light-color);
+    font-weight: 400;
+    outline: none;
+    padding:0.8rem;
+    border-radius: 0.5rem;
+    border:3px solid var( --badge-active-color);
+    width:20rem;
+
+}
+
```
Comment (Ankur9669) [inline Components/Input/input.css:54]: All the input boxes have same width, padding and border-radius but you have repeated them in 3 places. So try to creat a utility class.

**Review by Ankur9669 — CHANGES_REQUESTED** [review]

**Comment by varun-Tad** [comment]
Thanks for the review guys.I have made all the changes @Ankur9669 @gauravsinhaweb

**Pull request merged**